packages/backend/.swcrc

```diff
@@ -11,15 +11,19 @@
 			"decoratorMetadata": true
```

Comment:
these were needed because of some build problems, and after fixing them I found out that the upstream firefish made the same changes, so hopefully I was on the right track